lib/bin/lwm2m_carrier/lwm2m_carrier.rst

```diff
@@ -4,56 +4,72 @@ LwM2M carrier
+Your final device must go through certification by the carrier.
+
+The :ref:`lwm2m_carrier` sample demonstrates how to run this library in an application.
 
 Prerequisites
 *************
-The LwM2M carrier library has dependencies on Zephyr and the nRF Connect SDK.
-Check the release notes for the library to see which version of |NCS| it is built for.
-To integrate the library in an application, the application must be based on a supported nRF Connect SDK version.
+The LwM2M carrier library has dependencies on Zephyr and the |NCS|.
+See the :ref:`liblwm2m_carrier_changelog` of the library to check which version of |NCS| it is built for.
+To integrate the library into an application, the application must be based on a supported |NCS| version.
 
 Application integration
 ***********************
-To run the library in an application, it is mandatory to implement the application with the API of the library.
-The application has to keep the following values up to date, and the library with automatically report the updated values to the relevant server:
-Available Power Sources, Power Source Voltage, Power Source Current, Battery Level, Battery Status, Memory Free, Memory Total, Error Code, Device Type.
-The application also needs to implement an event handler and error handlers as shown in the client sample.
-The bsdlib_init function is called by the library because of the need to track modem update state.
+To run the library in an application, you must implement the application with the API of the library.
+
+The following values that reflect the state of the device must be kept up to date by the application:
+
+* Available Power Sources
+* Power Source Voltage
+* Power Source Current
+* Battery Level
+* Battery Status
+* Memory Total
+* Error Code
+* Device Type
 
-Features
-********
+The library automatically reports the updated values to the relevant server.
 
+The application must also implement an event handler and error handlers, as shown in the :ref:`lwm2m_client` sample.
+The ``bsdlib_init`` function is called by the library because of the need to track modem update state.
 
 Application limitations
 ***********************
 While running this module, the application has some limitations.
-The library depends on the LTE link controller to maintain connections to carrier servers when required, and the application has to run module.
+
+The library depends on the LTE link controller to maintain connections to carrier servers when required, so the application must run this module.
 Several system resources are used to maintain the device management platform connection.
 While running the module, the application cannot do the following:
 
-* Use the AT command module. This prevents the application from:
-    * subscribing to SMS from the modem,
-    * reading net registration status.
+* Use the AT command module.
+  This prevents the application from subscribing to SMS from the modem and reading the net registration status.
```

Comment:
Line 49, I have the same comment I made just above, this is not a limitation but a pseudo-dependency (the glue layer depends on this). There is nothing for the user to do here.
Line 50 is not correct, should be removed.

Reply:
removed both